README.md

```diff
@@ -2,3 +2,163 @@
+dracut: *** Constructing AuthenticAMD.bin ***
+dracut: *** Constructing GenuineIntel.bin ***
+dracut: *** Store current command line parameters ***
+dracut: *** Stripping files ***
+dracut: *** Stripping files done ***
+dracut: *** Creating image file '/boot/initramfs-6.5.5-300.fc39.x86_64.img' ***
+dracut: Using auto-determined compression method 'pigz'
+dracut: *** Creating initramfs image file '/boot/initramfs-6.5.5-300.fc39.x86_64.img' done ***
+```
+
+Excellent! Now, let's proceed to compare the size of the newly generated initramfs image
+with the existing ones in the filesystem.
+
+``` bash
+# uname -r
+6.5.5-300.fc39.x86_64
+
+# du -sh /boot/init*
+81M	 /boot/initramfs-0-rescue-285b2edb8ad94c7381215fd5720afd54.img
+34M	 /boot/initramfs-6.4.12-200.fc38.x86_64.img
+34M	 /boot/initramfs-6.5.5-200.fc38.x86_64.img
+
+13M  /boot/initramfs-6.5.5-300.fc39.x86_64.img     <- first image to load (storage drivers only)
+149M /boot/initoverlayfs-6.5.5-300.fc39.x86_64.img <- second image (extra kernel mods and files)
+```
+
+### Step 3 - Reboot to test
+
+To load the new generated **initramfs and initoverlayfs** images a reboot of the system is necessary.
+``` bash
+# reboot
+```
+
+### Step 4 - Validating the boot
+
+To validate whether the new image has been successfully loaded after the reboot, you can execute the following journalctl command and search for the keyword storage-init:
+
+``` bash
+# journalctl -r |  grep -i storage-init
+Oct 25 00:20:53 dell730.medogz.com storage-init: mount("/proc", "/initoverlayfs/proc", NULL, MS_MOVE, NULL)
+Oct 25 00:20:53 dell730.medogz.com storage-init: (stat("/initoverlayfs/proc", 0x7ffe85ccaa10) == 0) && 16 != 18)
+Oct 25 00:20:53 dell730.medogz.com storage-init: mount("/dev", "/initoverlayfs/dev", NULL, MS_MOVE, NULL)
+Oct 25 00:20:53 dell730.medogz.com storage-init: (stat("/initoverlayfs/dev", 0x7ffe85ccaa10) == 0) && 5 != 18)
+Oct 25 00:20:53 dell730.medogz.com storage-init: mount("/boot", "/initoverlayfs/boot", "ext4", MS_MOVE, NULL) 2 (No such file or directory)
+Oct 25 00:20:53 dell730.medogz.com storage-init: forked 368 fork_execlp
+Oct 25 00:20:53 dell730.medogz.com storage-init: fork_execlp("udevadm")
+Oct 25 00:20:53 dell730.medogz.com storage-init: forked 357 fork_execvp_no_wait
+Oct 25 00:20:53 dell730.medogz.com storage-init: fork_execvp_no_wait(0x16c4840)
+Oct 25 00:20:53 dell730.medogz.com storage-init: bootfs: {"UUID=b7eaec82-7c35-4887-becf-60ee7889624f", "bootfs UUID=b7eaec82-7c35-4887-becf-60ee7889624f"}, bootfstype: {"ext4", "bootfstype ext4"}, fs: {"(null)", "(null)"}, fstype: {"(null)", "(null)"}, udev_trigger: {"udevadm trigger --type=devices --action=add --subsystem-match=module --subsystem-match=block --subsystem-match=virtio --subsystem-match=pci --subsystem-match=nvme", "udev_trigger udevadm trigger --type=devices --action=add --subsystem-match=module --subsystem-match=block --subsystem-match=virtio --subsystem-match=pci --subsystem-match=nvme"}
```

Comment:
Ah you found my verbose logging :smile: ... Eventually I will turn many of these logs off (or make it configurable), it was useful at the start to get it working...